engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/util/StateConfiguration.scala

```diff
@@ -15,7 +14,9 @@ object StateConfiguration {
 
   }
 
-  case class RocksDBStateBackendConfig(checkpointDataUri: String,
+  case class RocksDBStateBackendConfig(enable: Boolean,
+                                       // deprecated - Not needed after Flink 1.13
+                                       checkpointDataUri: Option[String],
                                        dbStoragePath: Option[String],
```

Comment:
I wonder, maybe we  should also remove this one? And incremental checkpoints? Those can be configured on Flink cluster?

Reply:
We discussed that it won't be done in this change.